satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -273,24 +273,36 @@ read all (
     orderby asc project.name
 )
 
-// add is default, make composite key
 model project_payment (
-    key project_id
```

Comment:
Do we have a migration for this?

Reply:
yes.
Since we have no data, the simplest migration is to drop and recreate new table version